Where_Is_My_Wife/Assets/_Main/Assets/SaveSystem/DataSaveManager.cs

```diff
@@ -39,7 +39,7 @@ private void Save()
         private void Load()
         {
             string json = PlayerPrefs.GetString(SaveKey,null);
-            if (json == null)
+            if (json == null || json == "")
```

Comment:
Change to `string.IsNullOrEmpty(json)`, maybe you didn't know about it, we should use the helpers we have in our hands to avoid as much code as possible.